Userland/Utilities/fortune.cpp

```diff
@@ -84,7 +84,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     auto file = TRY(Core::File::open(path, Core::OpenMode::ReadOnly));
 
-    TRY(Core::System::pledge("stdio"));
```

Comment:
Why is the pledge gone?

Reply:
Diff is kinda hard to tell, but we have a pledge line above this:
`TRY(Core::System::pledge("stdio rpath"));`
This commit was to add rpath, so deleting this line is just keeping rpath in fortune.